src/components/customizableAside.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from './aside.module.css';
```

Comment:
@Sheyitofunmi Where is this file? I couldn't locate it locally 🤔

Reply:
I have uploaded it ma